### PR TITLE
Add Schwerin

### PR DESCRIPTION
**Pull request by Matthias84**
Permission by city and manual copied from http://marketing.schwerin.info/stadtmarketing/aufgaben/Flaeche_Maerkte.html

### CODE REVIEW

**Comment by johnjohndoe** [comment]
We [recently changed](https://github.com/wo-ist-markt/wo-ist-markt.github.io/commit/1b4f67f31d878b4f1d4fff40bca69fa71cfbc17d) the archtecture of the app. Please find up-to-date information in the contribution guideline. You now have to add the city to the `cities/cities.json`. In addition to that you forgot to edit the `README.MD`. Please update your commit as described earlier.

**Comment by Matthias84** [comment]
@johnjohndoe Thanks for the reminder. I hope the contribution is ok now?

File: cities/cities.json [hunk 1]
```diff
@@ -59,6 +59,10 @@
         "id": "rostock",
         "label": "Rostock"
     },
+    "schwerin": {
+        "id": "rostock",
```
Comment (johnjohndoe) [inline cities/cities.json]: copy paste error

**Comment by Matthias84** [comment]
Fixed the cities.json

File: README.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ A small visualization of the weekly markets in different cities. Available at: h
 |[Neu-Ulm][neu-ulm-wikipedia]|[City of Neu-Ulm][neu-ulm-markets]|
 |[Paderborn][paderborn-wikipedia]|[City of Paderborn][paderborn-markets]|
 |[Rostock][rostock-wikipedia]|[City of Rostock][rostock-markets]|
+|[Schwerin][schwerin-wikipedia]|[City of Schwerin][schwerin-markets]|
```
Comment (johnjohndoe) [inline README.md]: links are missing

**Comment by Matthias84** [comment]
Links fixed

**Comment by johnjohndoe** [comment]
:cat2: Thank you and welcome on the map (once again) with Schwerin!
I am looking forward to seeing a tweet from you with a screenshot and a link to the website.
Please also take a look at #38.

**Comment by dirkschumacher** [comment]
🎉

**Comment by johnjohndoe** [comment]
@Matthias84 Do you have time to see if the markets in Schwerin have been updated in the meantime? Here is another website with information: https://www.schwerin.de/kultur-tourismus/veranstaltungen/maerkte-verkaufsoffene-sonntage/
Please also take a look at updated the [contribution guide](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/CONTRIBUTING.md).

**Pull request merged**